README.md

```diff
@@ -106,7 +106,7 @@ go w.Work()
 ## Client
 
 ```go
-c, err := client.New("tcp4", "127.0.0.1:4730")
+c, err := client.NewNetClient("tcp4", "127.0.0.1:4730")
```

Comment:
I'm not sure why you change the constructor name here, it just breaks client compatibility.

Is there a reason to?

Reply:
The idea behind the change was that in our case "NewNetClient" represents a rather specialized case, and it would be less confusing to public than using "New" should be reserved for a general/universal case. This became more apparent after replacing NetConnected with NewClient which has become the generalized version where connectivity is handled by lambdas.